server/models/db/timeslot.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+const Tables = require('../../constants/tables.json');
+
+// This method returns the timeslot records stored in the timeslot table 
+// The method will return all timeslots in the timeslot table  
+async function getPossibleTimeslots() {
+
```

Comment:
Hi Peter,
I would recommend removing this blank line to reduce unnecessarily long file lengths.

Reply:
Removed!